web/lib/hooks/useSanityLoader.ts

```diff
@@ -12,6 +12,6 @@ export const useSanityLoader = (image: ImageWithAlt, maxWidth: number, aspectRat
         ? Math.round(width * aspectRatio)
         : Math.round(width * (cropped.height / cropped.width))
 
-      return imageUrlBuilder.width(width).height(height).auto('format').quality(70)
+      return imageUrlBuilder.width(width).height(height).auto('format').quality(100)
```

Comment:
I believe quality(100) is the default. So we could omit this if we want to go for highest quality